src/mono/sample/Android/AndroidSampleApp.csproj

```diff
@@ -10,6 +10,10 @@
     <ForceAOT Condition="'$(ForceAOT)' == ''">false</ForceAOT>
   </PropertyGroup>
 
+  <ItemGroup>
+      <RuntimeComponents Condition="'$(DiagnosticPorts)' != ''" Include="diagnostics_tracing" />
```

Comment:
indent

Reply:
What values can `$(DiagnosticPorts)` have? Is it a bool?

Reply:
> What values can `$(DiagnosticPorts)` have? Is it a bool?

This property is treated as a string and should carry values like:
`<address>:<port>:<mode>`
for example: `10.0.2.2:9000,suspend` or `10.0.2.2:9000,nosuspend`